### DISCUSSION TITLE
Would it be a good idea to customize the response after authorization?

### DISCUSSION THREAD

**Discussion by SeyTo** [zitadel/oidc#409] [General]
After a successful login, a redirect to the `redirect_uri` is performed and body has form of `oidc.AccessTokenResponse`. Would it be a good idea to let the `OPStorage` interface modify the response here as well?

**Comment by muhlemmer** [comment]
Sorry for no reply on this. I actually was looking for posts like this when I was defining https://github.com/zitadel/oidc/issues/440. The close notification triggered me to find this again.

We are in the process of defining a interface which should allow more customization in the future. Check out the issue and feel free to comment!
